tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/Examples/switchcase.lg

```diff
@@ -0,0 +1,8 @@
+﻿# greetInAWeek
```

Comment:
we should add a little bit more into this, and make the case inside more type of expressions, than just string

Reply:
Sure. I will put some more test cases in it.